src/components/home/index.js

```diff
@@ -1,3 +1,3 @@
-import components from './components';
+import * as components from './components';
```

Comment:
is ' * ' a wild card? what is it doing here
line 1

Reply:
Yes is a wild card, that says that all the imports from a specific value and assign it a name. Read over [es6 imports](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import) to understand what is happening here